app/main/lib/text_similarity.py

```diff
@@ -35,16 +35,29 @@ def get_document_body(body):
 def async_search_text(task, modality):
     return elastic_crud.get_async_presto_response(task, "text", modality)
 
+def sync_search_text(task, modality):
+    import code;code.interact(local=dict(globals(), **locals()))
```

Comment:
is code.interact a debugging line left in?

Reply:
indeed